code/datums/keybinding/keybinding.dm

```diff
@@ -9,6 +9,10 @@
 	/// Does this keybind apply regardless of any modifier keys (SHIFT-, ALT-, CTRL-)?
 	/// Important for movement keys, which need to still activate despite other "hold to toggle" bindings on the modifier keys.
 	var/any_modifier = FALSE
+	/// The key of the preference that we must have set to a specific value in order to show
```

Comment:
```suggestion
	/// The typepath of the preference that we must have set to a specific value in order to show
```

Reply:
It's a key, not a typepath but it makes more sense for this to be a type so I'll change it to a type